roles/core/repositories_server/readme.rst

```diff
@@ -17,17 +17,17 @@ Note that repositories structure follows a specific pattern:
 
                   Distribution    Version   Architecture    Repository
                         +             +       +               +
-                        |             +---+   |               |
-                        +-----------+     |   |      +--------+
-                                    |     |   |      |
-                                    v     v   v      v
-       /var/www/html/repositories/centos/7.6/x86_64/os
+                        |             +--+    |               |
+                        +-----------+    |    |    +----------+
```

Comment:
Same here, copy main doc part that explains the split between image and repos.

Reply:
Done